make/ha/docker-compose.tpl

```diff
@@ -0,0 +1,119 @@
+version: '2'
```

Comment:
Is there a reason we need this `ha/docker-compose.tpl`?

Reply:
Since the ha docker-compose will be different with standalone. we don't need some containers like mysql.